README.md

```diff
@@ -65,9 +66,12 @@ make gen-key
 - To try a simple rest method curl: `curl -X GET "http://localhost:8080/v1/admin/version"`. You should get the exchange version number as the response.  
 - When testing the exchange in an OpenShift Cluster the variables `EXCHANGE_IAM_ORG`, `EXCHANGE_IAM_KEY` and `EXCHANGE_MULT_ACCOUNT_ID` must be set accordingly.
 - A convenience script `src/test/bash/primedb.sh` can be run to prime the DB with some exchange resources to use in manually testing:
+- Before exporting below environment variables manually set them in `src/test/bash/primedb.sh`
+
```

Comment:
I don't understand this sentence. `primedb.sh` gets these variables from the environment at https://github.com/open-horizon/exchange-api/blob/b90546494646d231052e297d92120381d352e1b6/src/test/bash/primedb.sh#L40 . This way you don't need to change `primedb.sh`